### ISSUE TITLE
Class 'Cpdf' not found when using dompdf-develop with Composer

### ISSUE THREAD

**Issue by canfiax** [dompdf/dompdf#919]
I am trying to pull in the develop branch, using composer, like this:

    "require": {
		"dompdf/dompdf" : "dev-develop"
    }

And requiring like this:

	require 'vendor/autoload.php';
        use Dompdf/Dompdf;
	// disable DOMPDF's internal autoloader if you are using Composer
	define('DOMPDF_ENABLE_AUTOLOAD', false);
	define("DOMPDF_ENABLE_REMOTE", true);

	// include DOMPDF's default configuration
	require_once 'vendor/dompdf/dompdf/dompdf_config.inc.php';

But I am getting error:

     Fatal error: Class 'Cpdf' not found in /vendor/dompdf/dompdf/src/Dompdf/Adapter/CPDF.php on line 190

**Comment by bsweeney** [comment]
I'm still trying to work out all the kinks in the develop branch. Looks like Cpdf isn't appropriately hinted for composer so the file isn't loading. I'll look into that. In the meantime you can require Cpdf manually and you should be fine.

```php
<?php
require 'vendor/autoload.php';
require 'vendor/dompdf/dompdf/lib/Cpdf.php';

use Dompdf\Dompdf;
// disable DOMPDF's internal autoloader if you are using Composer
define('DOMPDF_ENABLE_AUTOLOAD', false);
define("DOMPDF_ENABLE_REMOTE", true);

// include DOMPDF's default configuration
require_once 'vendor/dompdf/dompdf/dompdf_config.inc.php';

// ...
```

I'm currently working on making dompdf_config.inc.php obsolete. The develop branch of dompdf  doesn't really use that file anymore (though there may be a few settings that still come into play). Configuration can be done (for the most part) by setting the options using the `set_option`/`set_options` methods.

```php
$dompdf->set_option('isRemoteEnabled', true);
$dompdf->set_options(
  array(
    'isRemoteEnabled' => true,
    'isCssFloatEnabled' => true
  )
);
```

After removing dompdf_config.inc.php this issue with Cpdf is (hopefully) the last bit we'll need so that we can go back to working on the actual functionality.

**Comment by bsweeney** [comment]
Have you tried the latest code in the develop branch? This issue might now be addressed.

**Comment by frost-nzcr4** [comment]
Latest develop works.

**Comment by bsweeney** [comment]
Thanks for the update @frost-nzcr4 :)

**Comment by bsweeney** [comment]
Ultimately fixed in commit da250b978bbd88dcca497dc761fe30b7d2958fb6.

**Comment by sjuvonen** [comment]
This does not seem to be fixed. I installed Dompdf using Composer as instructed in readme and included Composer's autoload script right at the top of my code.

`PHP Fatal error:  Uncaught Error: Class 'Cpdf' not found in /home/samu/tmp/vendor/dompdf/dompdf/src/Dompdf/Adapter/CPDF.php:190
Stack trace:
#0 /home/samu/tmp/vendor/dompdf/dompdf/src/Dompdf/CanvasFactory.php(66): Dompdf\Adapter\CPDF->__construct('letter', 'portrait', Object(Dompdf\Dompdf))
#1 /home/samu/tmp/vendor/dompdf/dompdf/src/Dompdf/Dompdf.php(260): Dompdf\CanvasFactory::get_instance(Object(Dompdf\Dompdf), 'letter', 'portrait')
#2 /home/samu/tmp/test.php(5): Dompdf\Dompdf->__construct()
#3 {main}
  thrown in /home/samu/tmp/vendor/dompdf/dompdf/src/Dompdf/Adapter/CPDF.php on line 190`
